service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -110,11 +110,21 @@ private RunSetDetailsResponse convertToRunSetDetails(RunSet runSet) {
   public ResponseEntity<RunSetListResponse> getRunSets(UUID methodId, Integer pageSize) {
     List<RunSet> runSets = runSetDao.getRunSets();
     List<RunSet> updatedRunSets = smartRunSetsPoller.updateRunSets(runSets);
-
-    List<RunSetDetailsResponse> runSetDetails =
-        updatedRunSets.stream().map(this::convertToRunSetDetails).toList();
-    RunSetListResponse response = new RunSetListResponse().runSets(runSetDetails);
-
+    RunSetListResponse response;
+
+    if (methodId != null) {
```

Comment:
🤦 

So obvious in hindsight!!

Reply:
^ I take this reaction to mean that, yes, we do want to handle missing arguments this way :)